src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -1,24 +1,73 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
 
+import java.util.stream.Collectors;
+
+import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.model.Model;
+import seedu.address.model.person.ListCommandPredicate;
 
 /**
- * Lists all persons in the address book to the user.
+ * Lists students in the address book to the user.
```

Comment:
Minor change, but maybe we could say "Lists students in the TA Toolkit"?

Reply:
Changed!